src/common/topology.js

```diff
@@ -574,6 +574,10 @@ var TopologyMap = function($http, topology) {
         .duration(500)
         .attr('r', r)
         .attr('_r', r);
+    })
```

Comment:
When the map updates every 30 seconds, it has to update the current scaling as well, otherwise it jumps back to the original size.